### DISCUSSION TITLE
Access Environment Variables inside Budibase Application

### DISCUSSION THREAD

**Discussion by CheshireNine** [Budibase/budibase#9055] [Help] [answered]
Is it possible to access **environment variables** or vars defined in budibase .env or docker_compose.yml config **inside application** similarly like local vars via default handler or JS:  {{ CurrentUser.Email}} or $("Device.Tablet")  ?

**Comment by melohagan** [comment, marked as answer]
Hey @CheshireNine 

This is not currently possible. There is an issue open for it here: https://github.com/Budibase/budibase/issues/4188
